packages/mastering/src/createMex.ts

```diff
@@ -120,6 +121,7 @@ export async function createMultiMex(
   }
   if (koeUpdate) {
     encryptAndSign(zipFile, 'koe-update.zip', keyAndIv, answersPrivateKey, koeUpdate)
+    sign(zipFile, 'koe-update.zip', answersPrivateKey, koeUpdate)
```

Comment:
I don't think this will work reliably. The `koeUpdate` stream should be cloned with something like https://www.npmjs.com/package/cloneable-readable so it can be safely passed to multiple consumers.

(`encryptAndSign` should probably use `cloneable-readable` as well rather than the homegrown passthrough hack)

Reply:
Good idea. Took `cloneable-readable` into use